src/6.7/6.7.2.0.md

```diff
@@ -0,0 +1,412 @@
+
+## System requirements
+
+* tested on PHP 8.2, 8.3 and 8.4
+* tested on MySQL 8 and MariaDB 11
+
+## Improvements
+
+### Performance Improvements
+
+Multiple performance bottlenecks were resolved, mainly addressing negative performance impact with growing data sets.
+
+#### Optimized OpenSearch Querying for product listing
+
+The way that sorting is handled when using OpenSearch was greatly improved in cases where translated fields where used for sorting, e.g. product names.
+
+Additionally, the queries were optimized to prevent nested queries, further improving performance. Overall, this can improve the OpenSearch query time by up to 85%.
+
+
+#### Dynamic category level loading
+
+The category tree's loading was changed so that the active levels for the sidebar navigation are loaded dynamically to always show available child categories, independently of the `navigation depth` setting in the sales channel config.
+
+This means that the `navigation depth` setting is now mainly responsible for the header navigation depth. 
+In cases where that setting was increased to support deeper nesting levels in the sidebar navigation, it can be set to a lower value again, which will positively affect performance.
+
+#### Optimized property group loading
+
+The way in which the property groups for listing filters are loaded was improved. This mainly involved optimizing the SQL queries to avoid pagination (which gets slower with more data) and perform fewer joins.
+Additionally, the sorting of the properties in the group was improved to use a more performant algorithm. 
+This will improve the performance of loading the property filters by up to 50%, especially with a large number of property filters.
+
+### Efficiency Improvements
+
+#### Lower memory footprint on data imports
+
+Memory usage, especially when importing a lot of custom fields, could be greatly reduced by reusing dynamic field definition objects and more efficient validator constraint caching.
+
+Additionally, a memory leak in HTMLSanitize was fixed, which increased memory usage the more sanitized fields were written in a single request. 
+
+This makes shopware more memory efficient, especially when writing bigger batches of data in a single request.
+
+#### More efficient administration OpenSearch indexing
+
+The indexing for the administration search indices on OpenSearch was improved to only reindex entities when search-relevant data was changed and not directly when any change happened to the entity.
```

Comment:
we can add real numbers to this that we observed in saas after this change:

>This can reduce the background work and amount of `AdminSearchIndexingMessages` dispatched to the queue by up to 95%